content/privacy/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Privacy Policy"
+no_menu: true
+---
+<div class="info" markdown="1">
+
+## Our website
+
+our IP address is stored for anti-abuse purposes and is not used in any other ways. We do not use any analytics service.
+
+(TODO: Mention that we do not use any CDNs at all?)
+
+### easyrpg.org
+
+Our main website does not collect any personal data.
+
+### blog.easyrpg.org
+
+### ci.easyrpg.org
+
+### communuity.easyrpg.org
+
+[Discourse Privacy Policy](https://community.easyrpg.org/privacy)
+
+### translate.easyrpg.org
+
+### wiki.easyrpg.org
+
+## Our software (Player, Editor, Tools, liblcf and OpenRTP)
+
+Our software does not access the internet. All data is stored locally. When reporting bugs we will ask you to submit some of this data manually.
+
+When you report a bug we ask you to submit the log file (``easyrpg_log.txt``) and the savegames (``SaveXX.lsd``). The log file contains information about the execution of the engine and does not contain any personal data. The savegames do not normally contain any personal data. Some games will ask you for personal data such as your name. This data is then available in the savegame.
+
+When EasyRPG Player crashes under Windows, a memory dump (``dmp``) is created. This can contain personal data such as your Windows user name.
+
+The typical way to report a bug is through GitHub. By using GitHub you agree to the [GitHub Privacy Policy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statemen). Alternatively, you can send us an email. If you use the Android Bug Report function, the log file and your savegames will be attached automatically.
+
+## External services
+
+### Source code
+
+For source code management and bug reports we use GitHub. When submitting code or bug reports you agree to the terms and conditions of GitHub: [GitHub Privacy Policy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statement).
+
+### Chat
+
+We use external service providers for the chat. The chat system is bridged. This means that all messages are visible across all networks. If you use the chat, you agree to the terms and conditions of these services. All chat messages are saved and are publicly accessible. We store time, username and the message: [Access the chatlogs](https://easyrpg.org/irc/log/).
+
+- [Discord Privacy Policy](https://discord.com/privacy)
+- [Libera Chat Privacy Policy](https://libera.chat/privacy)
+- [Matrix Privacy Policy](https://matrix.org/legal/privacy-notice)
+
+### Social Network
+
+We use external service providers to keep you up to date. By visiting these sites you agree to the terms and conditions:
+
+- [Facebook](https://facebook.com/privacy/policy)
+- [Instagram](https://help.instagram.com/155833707900388)
```

Comment:
Does this have no better link?
it references https://privacycenter.instagram.com/policy at least